cha/design/sync.tex

```diff
@@ -0,0 +1,97 @@
+\chapter{Synchronization}
+%What is the importance of synchronization to us
+    %hope for the best is not feasable
+    %To make psychoacoustic effects reality we need farily accurate manipulation depending on effect
+    %Audible sync is not the same as clock synchronization, a slight desync in clocks may actually cause psychoacoustic effects (precedence effect), essentially providing us with an acceptable threshold for desynchronization.
+Simply sending playback commands between devices and hoping they are received fast enough to not disrupt the audible synchronization, is not a reliable solution and as such synchronization is important to the application.\mnote{What a sentence}

```

Comment:
Overskriften introducere jo emnet synchronization, første linje nævner hvorfor det er relevant og den følgende linje nævner at vi skal bruge clock synkronisering for at kunne synkronisere lyd